src/lando/main/models/repo.py

```diff
@@ -38,6 +39,48 @@ class RepoError(Exception):
 class Repo(BaseModel):
     """Represents the configuration of a particular repo."""
 
+    # The path to the repository.
+    path: str
+    vcs: AbstractVcs
+
+    @property
+    def REJECTS_PATH(self) -> Path:
+        return self.vcs.REJECTS_PATH
```

Comment:
At this stage, the landing_worker still relies on this information, but I agree it may not be needed in the long run. For the moment, I prefer to limit the changes to scoping, and do heavier refactoring / deletions later, when I get a better sense of how Git support will influence the interface.